<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontUnitPanel</class>
 <widget class="QWidget" name="FrontUnitPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableViewUnit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1241</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>481</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>471</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widgetFilter" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="widgetTransfer" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxControl">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>330</y>
     <width>311</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QPushButton" name="pushButtonWorking">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Working</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEnableStep">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStep">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMonitor">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Monitor</string>
   </property>
   <widget class="QLabel" name="labelState">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNextState">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Next state</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
